packages/webviz-core/src/panels/ThreeDimensionalViz/commands/LaserScans.js

```diff
@@ -57,6 +70,7 @@ const laserScan = (regl: Regl) =>
           ? props.intensities
           : new Float32Array(props.ranges.length).fill(1),
       hitmapColor: (context, props) => new Array(props.ranges.length).fill(props.color || [0, 0, 0, 1]),
+      fallbackColor: (context, props) => new Array(props.ranges.length).fill(colorFromProps(props) || [0.5, 0.5, 1, 1]),
```

Comment:
If you decided to only support flat color, could use `uniform` instead of `attribute`.  Same for `hitmapColor`.  e.g. `overrideColor: (context, props) => props.settings?.overrideColor || [0.5, 0.5, 1, 1]`

Reply:
I did change both `hitmapColor` and `overrideColor` over to be `uniform` instead of `attribute`.

The actual color assignment now follows the pattern from [here](https://github.com/cruise-automation/webviz/blob/725912f321248aa8efab8c2bbad06edd2a86712f/packages/webviz-core/src/panels/ThreeDimensionalViz/commands/PointClouds/index.js#L251), where the default color is specified as a struct, that way I could fit it in one line easily. That `?.` operator is fancy, I didn't know that existed.